test/snippet/alphabet/all.cpp

```diff
@@ -13,7 +12,7 @@ int main()
 // dna4 my_letter{0};      // we want to set the default, an A
 // dna4 my_letter{'A'};    // we also want to set an A, but we are setting value 65
 
-// debug_stream << my_letter; // you expect 'A', but how would you access the number?
+// std::cout << my_letter; // you expect 'A', but how would you access the number?
```

Comment:
why not debug_stream?

Reply:
Because with the debug_stream this actually has a behaviour. And the example is about the hypothetical expectatations of users independet of SeqAn...